twitwi/normalizers.py

```diff
@@ -626,8 +632,8 @@ def normalize_tweet_v2(tweet, *, users_by_screen_name, places_by_id, tweets_by_i
         'text': unescape(text),
         'url': format_tweet_url(user['username'], tweet['id']),
         'hashtags': sorted(hashtags),
-        'mentioned_names': sorted(mentions),
-        'mentioned_ids': sorted(users_by_screen_name[name]['id'] for name in mentions),
+        'mentioned_names': sorted(mentions.keys()),
+        'mentioned_ids': sorted(mentions.values()),
```

Comment:
as a result, the name logic should apply here, ids should be listed in the order of mentions.keys()